vending-app/src/main/AndroidManifest.xml

```diff
@@ -93,6 +98,7 @@
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
+                <category android:name="android.intent.category.LAUNCHER"/>
```

Comment:
Please don't add LAUNCHER here for the regular build, this will clutter launchers of users with an unnecessary entry. We already add LAUNCHER category for Huawei devices via the Huawei flavor.